src/smt/command.cpp

```diff
@@ -1661,12 +1661,8 @@ void GetValueCommand::invoke(api::Solver* solver)
   {
     NodeManager* nm = solver->getNodeManager();
     smt::SmtScope scope(solver->getSmtEngine());
-    std::vector<Node> result;
-    for (const Expr& e :
-         solver->getSmtEngine()->getValues(api::termVectorToExprs(d_terms)))
-    {
-      result.push_back(Node::fromExpr(e));
-    }
+    std::vector<Node> result =
```

Comment:
I think we should update this method to use api::Term instead of Node. (In general, command.cpp should only use api::Term now I believe).

Reply:
Discussed offline. Will be addressed in a different PR.